community/bolt/src/main/java/org/neo4j/bolt/transport/TransportSelectionHandler.java

```diff
@@ -42,6 +43,7 @@
 {
     private static final String WEBSOCKET_MAGIC = "GET ";
     private static final int MAX_WEBSOCKET_HANDSHAKE_SIZE = 65536;
+    private static final int MAX_WEBSOCKET_FRAME_SIZE = 65536;
```

Comment:
Can `MAX_WEBSOCKET_FRAME_SIZE` and `MAX_WEBSOCKET_HANDSHAKE_SIZE` be made a single constant?

Reply:
That was my first intention but later on I decided to keep those distinct because `MAX_WEBSOCKET_HANDSHAKE_SIZE` applies to the pre-websocket communication where handshake occurs, whereas `MAX_WEBSOCKET_FRAME_SIZE` applies to websocket communication.